src/vsr/message_header.zig

```diff
@@ -663,10 +688,11 @@ pub const Header = extern struct {
         size: u32 = @sizeOf(Header),
         epoch: u32 = 0,
         view: u32,
-        version: u16 = vsr.Version,
+        protocol: u16 = vsr.Version,
+        release: u16 = 0,
```

Comment:
Comment that it is release of prepare and request

Reply:
Ahh, we _could_ set it to that, but this was actually a typo in my PR description; I intended to set this to zero.